libs/workspace/src/builders/design-tokens/tailwindcss/utils/generate-font-config.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DesignTokenSource } from '../../interfaces/design-token-source';
+
+/** Generates a config object for tailwinds font family configuration. */
+export function generateFontFamilies(
+  fontAliasSource: DesignTokenSource,
+): { [key: string]: string } {
+  return {
+    sans: fontAliasSource
+      .aliases!['font-family-sans-serif'].split(',')
+      .map((el) => el.trim()),
+    serif: fontAliasSource
+      .aliases!['font-family-serif'].split(',')
+      .map((el) => el.trim()),
+    mono: fontAliasSource
+      .aliases!['font-family-monospaced'].split(',')
+      .map((el) => el.trim()),
+  };
+}
+
+/** Generates a config object for tailwinds font size configuration. */
+export function generateFontSize(
+  typographyAliasSource: DesignTokenSource,
+): { [key: string]: string } {
+  const fontSizeConfig = {};
+  for (const [typoAliasName, typoAliasValue] of Object.entries(
+    typographyAliasSource.aliases!,
+  )) {
+    if (typoAliasName.includes('font-size')) {
+      // Get the correct sizing name from the alias name
+      // If there is no name left, it is the default value, which needs to be `base`
+      // for the tailwind config.
+      const transformedName =
+        typoAliasName.replace('typography--font-size-', '') || 'base';
+      fontSizeConfig[transformedName] = typoAliasValue;
+    }
+  }
+  return fontSizeConfig;
+}
+
+/** Generates a config object for tailwinds font weight configuration. */
+export function generateFontWeight(
+  typographyAliasSource: DesignTokenSource,
+): { [key: string]: string } {
+  const fontWeightConfig = {};
+  for (const [typoAliasName, typoAliasValue] of Object.entries(
+    typographyAliasSource.aliases!,
+  )) {
+    if (typoAliasName.includes('emphasis')) {
+      // Get the correct fontweight naming from the alias name
```

Comment:
NIT: It would be cool to have an example in here whats coming in and what you expect as a result after transformation. 
E.g. `typography--emphasis-regular` from the aliases would become `regular`

Reply:
Otherwise i'd have to know while reading what the structure of those aliases is